draft-irtf-cfrg-opaque.md

```diff
@@ -1392,33 +1393,32 @@ Output:
 
 Steps:
 1. dh1 = SerializePublicKey(sk1 * pk1)
-2. dh2 = SerializePublicKey(sk2 * pk2)
+2 dh2 = SerializePublicKey(sk2 * pk2)
 3. dh3 = SerializePublicKey(sk3 * pk3)
 4. Output concat(dh1, dh2, dh3)
 ~~~
 
-Using this shared secret, further keys used for encryption and authentication are
-computed using the following function.
-
 ~~~
-DeriveKeys(ikm, preamble)
+DeriveKeys(preamble, ikm)
```

Comment:
Why swap the parameter order?

Reply:
going back and forth, it just ended up like that, it was not intended. switching back